src/SolutionTemplate/UnoSolutionTemplate.VSIX/UnoPlatformPackage.cs

```diff
@@ -208,7 +217,29 @@ private async Task OnOpenedAsync(CancellationToken ct)
 						else
 						{
 							var toolsPath = System.IO.Path.Combine(unoNuGetPackage.InstallPath, "tools", "rc");
-							var asmPath = System.IO.Path.Combine(toolsPath, "Uno.UI.RemoteControl.VS.dll");
+
+							string GetRemoteControlVSPath()
+							{
+								if(!Environment.Is64BitProcess)
+								{
+									var files = new[] {
+										"Uno.UI.RemoteControl.VS.dll",
+										Path.Combine("16.0", "Uno.UI.RemoteControl.VS.dll")
+									};
+
+									return files
+										.Select(f => System.IO.Path.Combine(toolsPath, f))
+										.Where(File.Exists)
+										.FirstOrDefault();
+								}
+								else
+								{
+									return System.IO.Path.Combine(toolsPath, "17.0", "Uno.UI.RemoteControl.VS.dll");
+								}
```

Comment:
Keep positive first :)
```suggestion
								if(Environment.Is64BitProcess)
								{
									return System.IO.Path.Combine(toolsPath, "17.0", "Uno.UI.RemoteControl.VS.dll");
								}
								else
								{
									var files = new[] {
										"Uno.UI.RemoteControl.VS.dll",
										Path.Combine("16.0", "Uno.UI.RemoteControl.VS.dll")
									};

									return files
										.Select(f => System.IO.Path.Combine(toolsPath, f))
										.Where(File.Exists)
										.FirstOrDefault();
								}
```